docs/root/intro/what_is_envoy.rst

```diff
@@ -73,10 +73,10 @@ and logging for MongoDB connections.
 NOSQL datastore. Envoy :ref:`supports <arch_overview_dynamo>` L7 sniffing and statistics production
 for DynamoDB connections.
 
-**Service discovery:** :ref:`Service discovery <arch_overview_service_discovery>` is a critical
-component of service oriented architectures. Envoy supports multiple service discovery methods
-including asynchronous DNS resolution and REST based lookup via a :ref:`service discovery service
-<arch_overview_service_discovery_types_sds>`.
+**Dynamic configuration:** Envoy optionally consumes a layered set of :ref:`dynamic configuration
```

Comment:
Can you potentially merge the previous thing into this so it's something like "Service discovery and dynamic configuration?" I think the text in there about different types including DNS, etc. is important to mention?

Reply:
Done.